language/types/mixed.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?><!-- EN-Revision: 161dde4fe721309398dd324edbf02aec409f127b Maintainer: marcosmarcolin Status: ready --><!-- CREDITS: marcosmarcolin -->
+<sect1 xml:id="language.types.mixed">
+ <title>Misto</title>
```

Comment:
Esse aqui é o primeiro grande argumento a não traduzir os títulos aqui. Alguém que for pesquisar sobre mixed não vai pesquisar "misto" para encontrar esse material.